voucher/src/main/java/com/adyen/checkout/voucher/internal/ui/DefaultVoucherDelegate.kt

```diff
@@ -70,21 +75,54 @@ internal class DefaultVoucherDelegate(
             return
         }
 
-        _outputDataFlow.tryEmit(
-            VoucherOutputData(
-                isValid = true,
-                paymentMethodType = action.paymentMethodType,
-                downloadUrl = action.url
-            )
+        var viewType = VoucherComponentViewType.SIMPLE_VOUCHER
```

Comment:
You can make this a `val` and remove the `action.paymentMethodType?.let` statement just by allowing `VoucherPaymentMethodConfig.getByPaymentMethodType` to accept a null value